src/main/java/com/minecolonies/coremod/placementhandlers/BuilderIgnorePlacementHandler.java

```diff
@@ -0,0 +1,76 @@
+package com.minecolonies.coremod.placementhandlers;
+
+import com.ldtteam.structurize.placement.handlers.placement.IPlacementHandler;
+import com.ldtteam.structurize.util.BlockUtils;
+import com.ldtteam.structurize.util.PlacementSettings;
+import com.minecolonies.api.util.Log;
+import com.minecolonies.api.util.WorldUtil;
+import net.minecraft.core.BlockPos;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.JigsawBlock;
+import net.minecraft.world.level.block.StructureBlock;
+import net.minecraft.world.level.block.StructureVoidBlock;
+import net.minecraft.world.level.block.state.BlockState;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Collections;
+import java.util.List;
+
+import static com.ldtteam.structurize.placement.handlers.placement.PlacementHandlers.handleTileEntityPlacement;
+
+/**
+ * Handler for some specific blocks that we want only to paste/cost in the complete mode.
+ */
+public class BuilderIgnorePlacementHandler implements IPlacementHandler
+{
+    @Override
+    public boolean canHandle(@NotNull final Level world, @NotNull final BlockPos pos, @NotNull final BlockState blockState)
+    {
+        return blockState.getBlock() instanceof JigsawBlock || blockState.getBlock() instanceof StructureBlock || blockState.getBlock() instanceof StructureVoidBlock;
```

Comment:
... which makes this incorrect, because it shouldn't be ignoring them...

Structure Void should really be treated exactly identical to the tan placeholders.  Regular structure blocks shouldn't appear inside schematics at all (but if they do, it's probably reasonable to treat them like placeholders too).  Jigsaws are an entirely different animal, however.